md/prerequisites.md

```diff
@@ -60,9 +60,9 @@ Software required for BICI Add-on:
 
 Notes:
 1. BICI can be executed on Java 8. Java 9 and above are not yet supported. 
-1. All Subscription editions are supported.
-1. Archive must be activated
-1. BICI has its own connection pool to connect to Bonita engine database. Ensure database has available connections 
+2. All Subscription editions are supported.
```

Comment:
you don't need to change numbers, markdown will display them properly using `1.`for all items

Reply:
I know, it was done automatically from my editor. I think its ok to have directly the right number in the file